modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/authcode/UpdatingAuthorizationCodeState.java

```diff
@@ -64,6 +70,14 @@ public String getAccessToken() {
                                           "OAuth token for resource owner id " + delegate.getResourceOwnerId()
                                               + " has been invalidated");
     }
+    if (isClusterEnabled) {
```

Comment:
make this cluster agnostic, use a flag named properly for the behaviour it causes